tlatools/org.lamport.tlatools/test-model/CallGotoUnlabeledTest.tla

```diff
@@ -23,9 +23,9 @@ C:
 
 end algorithm *)
 \* BEGIN TRANSLATION - the hash of the PCal code: PCal-4786c31224fe8555dc7ad50128a92fc6
```

Comment:
The hashes should change

Reply:
I've committed my local diff, but shouldn't the hash change when pluscal code is changed? In this case, the tla+ translated code has changed not the input source

Reply:
The test specs/models might be outdated. There should be two hashes: one for PlusCal and one for the generated TLA+.

Reply:
See e.g. https://github.com/tlaplus/tlaplus/blob/master/tlatools/org.lamport.tlatools/src/pcal/Validator.java

Reply:
Retranslating the specifications and removing the old hashes should result in generating two new hashes.

Reply:
ack, let's sed (ah!) what I can do

Reply:
https://github.com/tlaplus/tlaplus/assets/88777/4919fde8-55c5-4d59-bf7b-044ed1eed178

Reply:
\* BEGIN TRANSLATION (chksum(pcal) = "5fd933e9" /\ chksum(tla) = "bff29291")

Reply:
I've pushed a new commit. I tried different routes but in the end replacing the "begin translation" and "end translation" strings and setting the cksum as empty string was the easiest way